examples/transportation/segment/road/road-path.yaml

```diff
@@ -0,0 +1,22 @@
+---
+id: overture:transportation:segment:1516
+type: Feature
+geometry:
+  type: LineString
+  coordinates: [[0, 0], [1, 1]]
+properties:
+  theme: transportation
+  type: segment
+  update_time: "2024-03-08T01:03:34-08:00"
+  version: 5
+  subtype: road
+  class: path
+  road:
+    surface:
+      - value: dirt
+    restrictions:
+      access:
+        - access_type: allowed
+          when: {mode: [foot]}
+        - access_type: denied
+          when: {mode: [bicycle]}
```

Comment:
nit: Experience has shown that the smaller and more focused the examples, the easier it is to refactor the schema since smaller examples result in fewer example files being touched by the refactor. So I would tend to drop everything optional (like the whole `road` property here).

Reply:
That's a good point @vcschapp ! I removed the additional properties to keep it simple.